src/iuse.cpp

```diff
@@ -5518,6 +5518,7 @@ int iuse::handle_ground_graffiti( player &p, item *it, const std::string &prefix
     string_input_popup popup;
     std::string message = popup
                           .title( prefix + " " + _( "(To delete, clear the text and confirm)" ) )
+                          .width( 85 )
```

Comment:
```suggestion
                          .description( prefix + " " + _( "(To delete, clear the text and confirm)" ) )
``` 
description is rendered above input string, so this would put newlines
![image](https://user-images.githubusercontent.com/54838975/195249962-a03d13d2-f03f-49de-b31f-2071779b0618.png)

Reply:
Nice!